domain/src/main/kotlin/no/nav/su/se/bakover/domain/visitor/LagBrevRequestVisitor.kt

```diff
@@ -528,20 +502,63 @@ class LagBrevRequestVisitor(
             )
         }
 
-    private fun vedtakIngenEndringIYtelse(vedtak: Vedtak.IngenEndringIYtelse): Either<KunneIkkeLageBrevRequest, LagBrevRequest> = hentPersonOgNavn(
-        fnr = vedtak.behandling.fnr,
-        saksbehandler = vedtak.saksbehandler,
-        attestant = vedtak.attestant,
-    ).map {
-        requestIngenEndring(
-            personOgNavn = it,
-            beregning = vedtak.beregning,
-            fritekst = when (val b = vedtak.behandling) {
-                is Revurdering -> b.fritekstTilBrev
-                else -> ""
+    private fun revurderingIngenEndring(revurdering: Revurdering, beregning: Beregning) =
+        hentPersonOgNavn(
+            fnr = revurdering.fnr,
+            saksbehandler = revurdering.saksbehandler,
+            attestant = FinnAttestantVisitor().let {
+                revurdering.accept(it)
+                it.attestant
             },
-            harEktefelle = vedtak.behandlingsinformasjon.harEktefelle(),
-            uføregrunnlag = vedtak.behandling.grunnlagsdata.uføregrunnlag
         )
-    }
+            .zip(hentGjeldendeUtbetaling(revurdering.sakId, LocalDate.now(clock)))
```

Comment:
Denne er jo litt spennende da.. Ønsker vi alltid å finne gjeldende utbetaling for dagens dato? For tilfeller hvor vi revurderer i nåtiden eller framover gir det sannsynligvis mening, men ved revurderinger tilbake i tid vil det vel fort bli misvisende? Dette problemet må sikkert sees i sammenheng hvordan vi ordlegger oss i brevet? Vil anta at revurderingens fra og med dato kanskje er det nærmeste vi kommer hvis vi bare skal vise 1 verdi? (ser at det er den datoen som er valgt nedenfor)

Reply:
Vi kom vel til at best-effort vi kan gjøre her er å bruke dagens dato ja, for det som kommuniseres i brevet er "dette er det beløpet du vil få videre". Dersom revurderingen er bakover i tid tenker jeg også at dette fortsatt er det nærmeste vi kommer.
Vi tror at det mest riktige egentlig hadde vært at brevet hadde mer detaljert informasjon. F.eks. "Det blir ingen endring i stønaden din. Dette er gjeldende utbetalingsplan" eller noe sånt.